utils/kata-manager.sh

```diff
@@ -957,8 +968,9 @@ handle_args()
 
 	local kata_version=""
 	local containerd_flavour="lts"
+	local kata_tarball=""
 
-	while getopts "c:dDfhk:lortT" opt "$@"
+	while getopts "c:dDfhkK:lortT" opt "$@"
```

Comment:
I think we lost a colon. Shouldn't this be...

```suggestion
	while getopts "c:dDfhk:K:lortT" opt "$@"
```